hibernate-core/src/main/java/org/hibernate/sql/results/graph/entity/internal/EntityJoinedFetchInitializer.java

```diff
@@ -32,7 +29,6 @@ public class EntityJoinedFetchInitializer extends AbstractEntityInitializer {
 
 	private final DomainResultAssembler<?> keyAssembler;
 	private final EntityValuedFetchable referencedFetchable;
-	private final boolean isEnhancedForLazyLoading;
```

Comment:
Interesting cleanup here! Love it - however I'm not sure I understand how this is a consequence of the holder idea

Reply:
It's a consequence of how the access to proxies changed.